source/read/retrieve.txt

```diff
@@ -0,0 +1,248 @@
+.. _scala-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, query, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to retrieve
+data from a MongoDB collection by using **read operations**. You can call the
+``find()`` method on a collection to retrieve documents that match a set of
+criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``companies`` collection in the ``sample_training``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your Scala application, create a ``MongoClient`` that connects to an Atlas cluster
+and assign the following value to your ``database`` and ``collection`` variables:
+
+.. literalinclude:: /includes/read/retrieve.scala
+    :language: scala
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+.. _scala-retrieve-find:
+
+Find Documents
+--------------
+
+To retrieve documents from a collection, use the ``find()`` method. This method
+takes a **query filter** parameter and returns an instance of the ``FindObservable`` class 
+from which you can access the query results. The ``FindObservable`` class also
+provides additional methods that you can chain to ``find()`` to modify its behavior,
```

Comment:
This might be more accurate

```suggestion
from which you can access the query results. The ``FindObservable`` class also
provides additional methods that you can chain to a ``FindObservable`` instance to modify its behavior,
```